### ISSUE TITLE
Allowing for specification of atomic quadrature size from input

### ISSUE THREAD

**Issue by wavefunction91** [merzlab/QUICK#13]
I couldn't find any mention of this type of functionality in the User Manual, so I apologize if this is already implemented. If it's not, would be useful to be able to specify the number of radial / angular quadrature points for the atomic quadratures from the input file to allow for users to toggle accuracy in the XC integration.

**Comment by miaoyipu** [comment]
I don't think it is supported, @Madu86 is the current maintainer, is this difficult to do so?

**Comment by merzlab** [comment]
Hi David, thanks for trying QUICK. We dont have this feature in v20.03 but will add down the road.

**Comment by Madu86** [comment]
@miaoyipu No. We can do this in a future release.
